### PR TITLE
updated caption

### PR DESCRIPTION
**Pull request by Deepika-tmna**

### CODE REVIEW

File: _slides/03-Deepika-tmna.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Welcome!!!
```
Comment (gusshawstewart) [inline _slides/03-Deepika-tmna.md:6]: This is a great change! 😄

File: _slides/03-Deepika-tmna.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (gusshawstewart) [inline _slides/03-Deepika-tmna.md]: Have you thought about doing it like this:
```suggestion
title: "Welcome to our slide deck! it is the best slide deck"
```

**Review by gusshawstewart — APPROVED** [review]
LGTM 🚢 🏅

File: _slides/03-Deepika-tmna.md (hunk 1 above)
Comment (Deepika-tmna) [reply _slides/03-Deepika-tmna.md:6]: Thank you 👍

**Pull request merged**